build_modules/lib/src/errors.dart

```diff
@@ -102,3 +104,36 @@ Future<String> _missingImportMessage(
   var lineInfo = parsed.lineInfo.getLocation(import.offset);
   return '`$import` from $sourceId at $lineInfo';
 }
+
+/// An [Exception] that is thrown when there are some unsupported modules.
+class UnsupportedModulesException implements Exception {
```

Comment:
I think this one is also readable as `UnsupportedModules` and the suffix is noise.